INSTALL.md

```diff
@@ -81,6 +60,9 @@ cmake -DENABLE_DRAFTS=OFF -DCPPZMQ_BUILD_TESTS=OFF ..
 sudo make install
 ```
 
+And then pass `-DCPPZMQ_BASE=/usr/local` to cmake or use
+`CXXFLAGS="-I /usr/local/include" configure ..` when compiling via the tango source distribution.
```

Comment:
I would split `CXXFLAGS="-I /usr/local/include" configure ..` in 2 lines too to make it less confusing:

And then pass `-DCPPZMQ_BASE=/usr/local` to cmake or use
```mkdir build
cd build
CXXFLAGS="-I /usr/local/include"
configure ..
```
 when compiling via the tango source distribution.

Reply:
That is something different. Here you are setting the environment variable CXXFLAGS for the whole shell. And in my version you are just setting it for the configure invocation.

How about

```
mkdir build
cd build
configure CXXFLAGS="-I /usr/local/include" ..
```

which passes CXXFLAGS to configure as argument?
Do out-of-tree builds work with configure?

Reply:
> That is something different. Here you are setting the environment variable CXXFLAGS for the whole shell. And in my version you are just setting it for the configure invocation.

Sorry, you're right...

 > How about
> 
> ```
> mkdir build
> cd build
> configure CXXFLAGS="-I /usr/local/include" ..
> ```
> 
> which passes CXXFLAGS to configure as argument?

This should work too.

> Do out-of-tree builds work with configure?

Yes. Out-of-tree builds work with configure

Reply:
Okay, I'll change it.